adminer/indexes.inc.php

```diff
@@ -52,14 +53,15 @@
 					&& array_values($existing["columns"]) === $columns
 					&& (!$existing["lengths"] || array_values($existing["lengths"]) === $lengths)
 					&& array_values($existing["descs"]) === $descs
+					&& (array_key_exists("method", $existing) && $existing["method"] === $indexMethod)
```

Comment:
You don't need `array_key_exists` if you initialize `$index_method` with `null`.